boa/src/builtins/array/mod.rs

```diff
@@ -233,6 +233,7 @@ impl Array {
         array.ordinary_define_own_property(
             "length".into(),
             PropertyDescriptor::builder()
+                // todo: check if we can refactor JsValue to hold u32
                 .value(length as f64)
```

Comment:
```suggestion
                .value(length)
```

It should be able to convert `u32` to `JsValue` 

Edit: https://github.com/boa-dev/boa/blob/master/boa/src/value/conversions.rs#L58-L67

Reply:
Nice! Let me change it then.